container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/kubernetes_kurtosis_backend_api_container_functions.go

```diff
@@ -42,6 +42,9 @@ const (
 
 var noWait *port_spec.Wait = nil
 
+// TODO add support for passing toleration to APIC
```

Comment:
is this TODO for the next iterations?

Reply:
checking just in case

Reply:
Yes! https://github.com/kurtosis-tech/kurtosis/issues/2073